app/controllers/laa_references_controller.rb

```diff
@@ -1,42 +1,49 @@
 # frozen_string_literal: true
 
 class LaaReferencesController < ApplicationController
-  include DefendantSearchable
-
-  before_action :set_defendant_asn_or_nino_if_required,
-                :load_and_authorize_search,
+  before_action :set_defendant_uuid_if_required,
+                :authorize_defendant_search,
```

Comment:
[ditto](https://github.com/ministryofjustice/laa-court-data-ui/pull/231/files#r464481290)

Reply:
👍 amended to load_and_authorize_defendant_search